components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,336 @@
++++
+title = "ServiceNow Integration"
+
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "ServiceNow Integration"
+    parent = "automate/reference"
+    identifier = "automate/reference/servicenow_integration.md ServiceNow Integration"
+    weight = 40
++++
+
+
```

Comment:
While there is a lot of content, but it seems flow breaks in middle and we go deep in explaining advanced concepts. Our recommendation will be to have this structure: 
- Overview 
- Installation
- Configuration
- Validation of results -> Navigation, client and compliance tables
- Benchmarking
- Advanced concepts  -> Transform Maps, Script Actions, Events 
- Customisation
- Troubleshooting Guide

Reply:
done